README.md

```diff
@@ -30,53 +31,75 @@ For more information:
 
 > `npm install --save fluture`
 
+### In ES5 or older environments
+
+Fluture depends on these functions being present: `Object.create`,
+`Object.assign` and `Array.isArray`. You may need to polyfill one or more.
```

Comment:
It would be nice to have these three function names link to the corresponding MDN documentation.